packages/react/src/textarea/Textarea.css.ts

```diff
@@ -0,0 +1,53 @@
+import { recipe } from "../vanilla-extract";
+
+export const textAreaBoxRecipe = recipe({
+  base: {
+    background: "transparent",
+    flexGrow: "1",
+    fontFamily: "sans",
```

Comment:
we don't need this

Reply:
keeping flexGrow as the middle part of textarea should expand if needed, bhai